libpod/container.go

```diff
@@ -284,13 +284,14 @@ type ContainerConfig struct {
 	// NetNsCtr conflicts with the CreateNetNS bool
 	// These containers are considered dependencies of the given container
 	// They must be started before the given container is started
-	IPCNsCtr    string `json:"ipcNsCtr,omitempty"`
-	MountNsCtr  string `json:"mountNsCtr,omitempty"`
-	NetNsCtr    string `json:"netNsCtr,omitempty"`
-	PIDNsCtr    string `json:"pidNsCtr,omitempty"`
-	UserNsCtr   string `json:"userNsCtr,omitempty"`
-	UTSNsCtr    string `json:"utsNsCtr,omitempty"`
-	CgroupNsCtr string `json:"cgroupNsCtr,omitempty"`
+	IPCNsCtr     string `json:"ipcNsCtr,omitempty"`
+	MountNsCtr   string `json:"mountNsCtr,omitempty"`
+	NetNsCtr     string `json:"netNsCtr,omitempty"`
+	PIDNsCtr     string `json:"pidNsCtr,omitempty"`
+	UserNsCtr    string `json:"userNsCtr,omitempty"`
+	UserNsKeepId bool
```

Comment:
This should be elsewhere, have a JSON tag, and a comment describing what it does